sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_models.py

```diff
@@ -229,6 +229,18 @@ def updated_on(self):
         """
         return self._attributes.updated if self._attributes else None
 
+    @property
+    def recoverable_days(self):
+        # type: () -> Optional[int]
+        """The number of days the certificate is retained before being deleted from a soft-delete enabled Key Vault.
```

Comment:
If this is consistent already, leave it. But it would really be "soft delete-enabled" when used as an adjective. I changed all these in .NET. Again, though, just leave it if this is consistent.